functions/Remove-DbaDbView.ps1

```diff
@@ -0,0 +1,118 @@
+function Remove-DbaDbView {
+    <#
+    .SYNOPSIS
+        Removes a database view from database(s) for each instance(s) of SQL Server.
+
+    .DESCRIPTION
+        The Remove-DbaDbView removes view(s) from database(s) for each instance(s) of SQL Server.
+
+    .PARAMETER SqlInstance
+        The target SQL Server instance or instances. This can be a collection and receive pipeline input to allow the function to be executed against multiple SQL Server instances.
```

Comment:
No, no pipeline input for SqlInstance.

Reply:
Yes, all of our commands should accept input from a string or `Get-Content serverlist.txt | Verb-Dba...`

This should not be removed and is the template description used across all of our public functions.